### PR TITLE
chore: Set up CI to release

### PR DESCRIPTION
**Pull request by tumido**
SSIA

### CODE REVIEW

**Comment by tumido** [comment]
cc @HumairAK @4n4nd @harshad16 

@harshad16 can you please check if I haven't forgotten about anything in here?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+*    @HumairAK @tumido @4n4nd
```
Comment (harshad16) [inline .github/CODEOWNERS:1]: this no needed from the ci side, OWNERS file at root is enough

**Review by harshad16 — COMMENTED** [review]
all lgtm
thanks :+1:

**Comment by goern** [comment]
/hold

you might want to cherry-pick

https://github.com/thoth-station/template-project/commit/31d34d9a5ce0846494d0ccdc0e869d1b51310d63
https://github.com/thoth-station/template-project/commit/fc530bb9e8f872b2d514ab6e6976feef39e194ef

**Comment by tumido** [comment]
Thank you @goern, done :+1: 

/unhold

**Review by harshad16 — COMMENTED** [review]
/lgtm

**Review by tumido — COMMENTED** [review]
/approve

**Pull request merged**